tincr/io/library/genFamilyInfo.tcl

```diff
@@ -176,9 +179,10 @@ proc processSite {s type compatible is_alt fo} {
 		
 		#print the bel information 
 		set tmpname [suffix $b "/"]
+		set tmptype [prefix $b "/"]
 		puts $fo "        <bel>"
 		puts $fo "          <name>$tmpname</name>"
-		puts $fo "          <type>$tmpname</type>"
+		puts $fo "          <type>$tmptype</type>"
```

Comment:
Why not use Tincr's [`tincr::bels get_type`](https://github.com/byuccl/tincr/blob/master/tincr/cad/device/bels.tcl#L113) method?